### ISSUE TITLE
Module ‘Control.Monad.Error.Class’ does not export ‘Error(..)’

### ISSUE THREAD

**Issue by sjakobi** [haskell/mtl#95]
I'm experimenting with `mtl` `master` and encountered this issue with `tls`:

```
Network/TLS/ErrT.hs:19:35: error:
    Module ‘Control.Monad.Error.Class’ does not export ‘Error(..)’
   |
19 | import Control.Monad.Error.Class (Error(..))
   |                                   ^^^^^^^^^
```

I didn't see this mentioned in the changelog. Possibly an oversight?!

**Comment by chessai** [comment]
So, we need to provide a changelog entry that Error(T) exports were removed, not just their instances. Yes, this was an oversight in the changelog.

**Comment by sjakobi** [comment]
Fixed in #100. Thanks again, @chessai! :)
